.github/workflows/deploy-public-build.yml

```diff
@@ -0,0 +1,37 @@
+# Adapted from https://github.com/marketplace/actions/deploy-pr-preview.
+name: Deploy Public Build
+
+on:
+  push:
+    branches:
+      - main
+
+concurrency: pages-build-deployment-${{ github.ref }}
+
+jobs:
+  pages-build-deployment:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: "18"
+
+      - name: Install and Build
+        run: npm ci && npx vite build --base=https://allen-cell-animated.github.io/nucmorph-colorizer/
```

Comment:
is it necessary to hardcode the url here? oh well

Reply:
I can try it without, it was only necessary for the preview subfolders.